core/templates/dev/head/pages/exploration-editor-page/settings-tab/settings-tab.directive.html

```diff
@@ -4,7 +4,7 @@ <h3>Basic Settings</h3>
     <div class="row">
       <div class="col-lg-12 col-md-12 col-sm-12">
         <div ng-if="$ctrl.EditabilityService.isEditable()">
-          <div role="form" class="form-horizontal">
+          <div role="form" class="form-horizonta">
```

Comment:
I'm pretty sure this is wrong -- please proofread your PR very carefully.

Reply:
Done Updated this